src/components/DataTable/DataTable.tsx

```diff
@@ -75,7 +75,12 @@ export class DataTable extends React.PureComponent<
     } = this;
     let collapsed = false;
     if (table && scrollContainer) {
-      collapsed = table.scrollWidth > scrollContainer.clientWidth;
+      const headerCells = table.querySelectorAll(
```

Comment:
Can we just `table.querySelector` to get the first header cell since we are only using the first element below (`headerCells[0].offsetWidth`)?